ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-presubmits.yaml

```diff
@@ -81,22 +81,10 @@ presubmits:
         env:
         - name: TEST_FOCUS
           value: Suite:openshift/conformance/parallel
-        - name: TEST_FOCUS_SERIAL
-          value: Suite:openshift/conformance/serial/minimal
-        - name: TEST_SKIP
-          value: .*((The HAProxy router should set Forwarded headers appropriately)|(oc
-            new-app should succeed with a --name of 58 characters)|(process valueFrom
-            in build strategy environment variables should successfully resolve valueFrom
-            in docker build environment variables)|(prune builds based on settings
-            in the buildconfig should prune failed builds based on the failedBuildsHistoryLimit
-            setting)|(Prometheus when installed to the cluster should start and expose
-            a secured proxy and unsecured metrics)|(Prometheus when installed to the
-            cluster should start and expose a secured proxy and verify build metrics)|(templateservicebroker)|(deploymentconfigs
-            with multiple image change triggers \[Conformance\] should run a successful
-            deployment with multiple triggers)|(templateinstance readiness test should
-            report failed soon after an annotated objects has failed)|(s2i build with
-            a root user image should create a root build and pass with a privileged
-            SCC)).*
+        - name: JOB_NAME_SAFE
+          value: e2e-azure
```

Comment:
This needs to be unique per job and needs to match the base path of `--template` so `e2e-azure-conformance`